### DISCUSSION TITLE
Restricting Azimuth to 0-359 and Inclination 0-90 fields (ArcPro->FieldMaps)?

### DISCUSSION THREAD

**Discussion by JoshuaKeeley** [usgs/gems-tools-pro#110] [General]
I'm trying to use Attribute Rules to restrict Azimuth to 0-359 and Inclination to 0-90. Anyone know an Arcade expression to help me do that? Hopefully the restricted values will carry over to FieldMaps. Thanks!

**Comment by JoshuaKeeley** [comment]
For Azimuth field: return ($feature.Azimuth >= 0 && $feature.Azimuth <= 359);

For Inclination field: return ($feature. Inclination >= 0 && $feature.Inclination <= 90);

Set triggers to Insert and Update, and add an error message.
![Azi_Inc_AttributeRules](https://github.com/DOI-USGS/gems-tools-pro/assets/135344858/2d780ffa-1dee-4aab-83ba-86c5a4fa92f0)

**Comment by dmirandaUSGS** [comment]
Looks like you figured out the Arcade expression, before I could reply!

For ArcGIS Online Field Maps, attribute rules do not carry over. However, you should be able to set a valid numeric range directly in the Field Maps Form, see `Create Range` on an example field below:

![image](https://github.com/DOI-USGS/gems-tools-pro/assets/119333158/28637002-8b83-4dfc-93cf-dadbec3d4857)

**Reply by JoshuaKeeley** [reply]
Thanks for the quick reply and for the Field Maps form info. I will try that.
